onnxruntime/python/tools/transformers/convert_to_onnx.py

```diff
@@ -138,12 +146,36 @@ def main():
         assert not args.output.endswith('.onnx'), "output shall be a directory for --use_external_data_format"
 
     model_class = MODEL_CLASSES[args.model_class][0]
-    model_type = "beam_search_step" if args.model_class == "GPT2LMHeadModel_BeamSearchStep" else "default"
+    if args.model_class == "GPT2LMHeadModel_BeamSearchStep":
+        model_type = "beam_search_step"
+    elif args.model_class == "GPT2LMHeadModel_BeamSearchStepConfiguration":
```

Comment:
How about use name "GPT2LMHeadModel_BeamSearchStepV2"?

Reply:
Good suggestion! Maybe "GPT2LMHeadModel_ConfigurableOneStepSearch" is a even better name?

Reply:
Good suggestion! Maybe "GPT2LMHeadModel_ConfigurableOneStepSearch" is a even better name?

Reply:
GPT2LMHeadModel_ConfigurableOneStepSearch is good

Reply:
Thanks! It is done